music21/musicxml/m21ToXml.py

```diff
@@ -4576,7 +4575,7 @@ def chordSymbolToXml(self, cs):
           <kind>suspended-fourth</kind>
           <degree>
             <degree-value>9</degree-value>
-            <degree-alter />
+            <degree-alter>0</degree-alter>
```

Comment:
degree-alter can be empty, or at least I have seen it empty in the wild -- does that still work?

Reply:
thanks!